src/bitcoinForks/index.ts

```diff
@@ -268,7 +268,7 @@ export default class BitcoinWallet implements Partial<IWallet> {
       chainIndex +
       addressIndex +
       contractDummyPadding +
-      '00'
+      intToUintByte(0, 64)
```

Comment:
Won't this break backward compatibility? I would suggest doing something similar to this https://app.clickup.com/t/37308523/CSI-1260

Reply:
yes it will break. will need to bump SDK version.

Reply:
Bumping sdk won't be enough also need to use featureMap in communications and modify the code accordingly

Reply:
Changes added

Reply:
@TejasvCypherock good catch